### PR TITLE
fix typings

### PR DESCRIPTION
**Pull request by tanjunior**
Resolving some typecheck warnings/errors that I noticed when using the components in my project.

### CODE REVIEW

File: apps/www/components/ui/use-toast.ts [hunk 1]
```diff
@@ -135,7 +135,7 @@ function dispatch(action: Action) {
   })
 }
 
-interface Toast extends Omit<ToasterToast, "id"> {}
+type Toast = Omit<ToasterToast, "id">
```
Comment (jocarrd) [inline apps/www/components/ui/use-toast.ts:138]: I love this simplification

**Review by jocarrd — APPROVED** [review]

**Comment by tanjunior** [comment]
@jocarrd made it up to date with main branch. Appreciate if you could approve again.

**Review by jocarrd — APPROVED** [review]

**Review by jocarrd — APPROVED** [review]

**Comment by tanjunior** [comment]
@jocarrd Made a new commit. ran prettier since the format check failed.

**Review by jocarrd — APPROVED** [review]

**Review by shadcn — APPROVED** [review]
Thank you.

**Comment by tanjunior** [comment]
> Thank you.

thanks for merging PR. happy to be of help.

**Pull request merged**